public/views/swagger.html

```diff
@@ -33,31 +37,56 @@
   <body>
     <div id="swagger-ui"></div>
 
-    <script nonce="[[.Nonce]]" src="https://unpkg.com/swagger-ui-dist@4.3.0/swagger-ui-bundle.js" charset="UTF-8" integrity="sha384-BGJ5JzR5LEl4ETmxXXlZtXtMWj3uQ9jj9/OHe3yrn5rrtAyLOz1SyyzwMfuwZgPc" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-    <script nonce="[[.Nonce]]" src="https://unpkg.com/swagger-ui-dist@4.3.0/swagger-ui-standalone-preset.js" charset="UTF-8" integrity="sha384-AWSfISmlS8fS336GXRkpL0Uv6EbCpsFfXDUwmklhbb3SctGSuvXWBcbjERjgf/e4" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script
+      nonce="[[.Nonce]]"
+      src="https://unpkg.com/swagger-ui-dist@4.3.0/swagger-ui-bundle.js"
+      charset="UTF-8"
+      integrity="sha384-BGJ5JzR5LEl4ETmxXXlZtXtMWj3uQ9jj9/OHe3yrn5rrtAyLOz1SyyzwMfuwZgPc"
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
+    <script
+      nonce="[[.Nonce]]"
+      src="https://unpkg.com/swagger-ui-dist@4.3.0/swagger-ui-standalone-preset.js"
+      charset="UTF-8"
+      integrity="sha384-AWSfISmlS8fS336GXRkpL0Uv6EbCpsFfXDUwmklhbb3SctGSuvXWBcbjERjgf/e4"
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
     <script nonce="[[.Nonce]]">
-    window.onload = function() {
-      // Begin Swagger UI call region
-      const ui = SwaggerUIBundle({
-        url: "/public/api-merged.json",
-        dom_id: '#swagger-ui',
-        deepLinking: true,
-        presets: [
-          SwaggerUIBundle.presets.apis,
-          SwaggerUIStandalonePreset
-        ],
-        plugins: [
-          SwaggerUIBundle.plugins.DownloadUrl
-        ],
-        layout: "StandaloneLayout",
-        filter: true,
-        tagsSorter: "alpha",
-        tryItOutEnabled: true
-      });
-      // End Swagger UI call region
+      window.onload = async function () {
+        const urls = [
+            {name: 'Grafana API', url: "/public/api-merged.json"},
+        ];
+        try {
+          const rsp = await fetch('/openapi/v3');
+          const apis = await rsp.json();
+          for (const [key, value] of Object.entries(apis.paths)) {
+            const parts = key.split('/');
+            if (parts.length == 3) {
+              urls.push({name: `${parts[1]}/${parts[2]}`, url: value.serverRelativeURL});
+            }
+          }
+        }
+        catch (err) {
+          console.warn('Error loading k8s apis');
+        }
```

Comment:
^^ lines 58 to 75 are the real change.  This adds each k8s service to the dropdown